projects/storefrontlib/src/cms-components/product/product-details/product-summary/product-summary.component.html

```diff
@@ -1,51 +1,55 @@
-<ng-template
-  [cxOutlet]="outlets.RATING"
-  [cxOutletContext]="{ product: product }"
->
-  <div class="rating">
-    <cx-star-rating
-      [rating]="product?.averageRating"
-      [disabled]="true"
-    ></cx-star-rating>
-    <div class="count">({{ product?.numberOfReviews }})</div>
-    <a class="btn-link" *ngIf="reviewsTabAvailable" (click)="showReviews()">{{
-      'productSummary.showReviews' | cxTranslate
-    }}</a>
-  </div>
-</ng-template>
-<ng-template
-  [cxOutlet]="outlets.TITLE"
-  [cxOutletContext]="{ product: product }"
->
-  <!-- <div class="name">{{ product?.name }}</div> -->
-  <div class="code">
-    {{ 'productSummary.id' | cxTranslate }} {{ product?.code }}
-  </div>
-</ng-template>
+<ng-container *ngIf="(product$ | async) as product">
+  <ng-template
+    [cxOutlet]="outlets.RATING"
```

Comment:
Did we not say INTRO fro both rating and id? Since there's no longer a title, we shouldn't use TITLE.

Reply:
I create an INTRO flex component but there is no mapping SPA for it at the moment. This could be a follow up task

Reply:
That's fine, I only meant the outlet name here.